src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb

```diff
@@ -0,0 +1,1412 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "94a47c81",
+   "metadata": {},
+   "source": [
+    "# Using the SPARQL1.1 query language\n",
+    "\n",
+    "Working with Semantic Web knowledge graphs can be a daunting prospect for a newcomer.  Having lots of new acronyms and terms to learn and understand – like RDF, SPARQL and knowledge graph – can been seen as a barrier of entry for these powerful technologies. \n",
+    "\n",
+    "In reality, Semantic Web knowledge graph databases are a very easy and intuitive way to understand, record and traverse data. In our opinion, much easier to use than relational databases. \n",
```

Comment:
I suggest rewording this a little from "In our opinion...." to something like "This is often easier to do using graph databases, and graph query languages, as they are optimized for these types of operation"